draft-ietf-rats-eat.md

```diff
@@ -621,9 +645,13 @@ against physical or electrical attacks against the device itself.
 It is assumed any potential attacker has captured the device and can 
 disassemble it. Example include TPMs and Secure Elements.
 
+The entity should claim the highest security level it achieves and no higher.
+This set is NOT extensible so as to provide a clear common interoperable description of security level to the relying party.
+If a particular implementation considers this claim to be inadequate, they should 1) omit the claim out, 2) define their own proprietary claim or 3) both.
```

Comment:
I would simply say:
_"If a particular implementation considers this claim to be inadequate, they should omit it and define their own proprietary claim instead."_